### ISSUE TITLE
Visual mode, vi bindings: Most bindings are unconfigured

### ISSUE THREAD

**Issue by johanwiden** [atlas-engineer/nyxt#2896]
**Is your feature request related to a problem? Please describe.**
In Nyxt 3-pre-release-5 most key bindings for vi visual mode, are missing.

**Describe the solution you'd like**
Please document at least in which keymap they should be bound.
Also, if possible, please provide default vi key bindings, for visual mode.
I personally like the functionality that the Google Chrome extension Vimium provides, for this case.

**Describe alternatives you've considered**

**Additional context**

**Comment by aartaka** [comment]
Yes, you're right, VI ones are missing. We'll be glad to have some help with these, given that most of the team uses Emacs bindings. Can you list the bindings that are required in visual mode, or direct us to the reference for that?

**Comment by johanwiden** [comment]
Wiki page for vimium visual mode: https://github.com/philc/vimium/wiki/Visual-Mode
I consider having keys similar to vimiums
  v c w y
and
  h j k l
essential. v enters visual mode. c can then be used to change to caret mode.
In visual mode text moved over is marked. In caret mode one moves without marking text.
w moves forward by word.
h j k l moves by character left down up right.
y copies the marked text to the clipboard, and exits visual mode.
One can also exit without copying, by using something like the escape key.

I should also point out that in Google Chrome with vimium one can use the mouse to indicate the position where visual mode should start.

**Comment by CorruptedVor** [comment]
and pressing y twice should yank (copy to clipboard) from within visual mode

**Comment by CorruptedVor** [comment]
> v enters visual mode. c can then be used to change to caret mode.

change to caret mode exclusively within visual mode, or from normal mode as well?
